[2023-08-01 04:47:37,272] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_api_marvel_hacia_aws_redshift_multitask.extraer_data scheduled__2023-08-01T04:45:35.277444+00:00 [queued]>
[2023-08-01 04:47:37,298] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_api_marvel_hacia_aws_redshift_multitask.extraer_data scheduled__2023-08-01T04:45:35.277444+00:00 [queued]>
[2023-08-01 04:47:37,298] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-08-01 04:47:37,298] {taskinstance.py:1377} INFO - Starting attempt 1 of 11
[2023-08-01 04:47:37,298] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-08-01 04:47:37,360] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): extraer_data> on 2023-08-01 04:45:35.277444+00:00
[2023-08-01 04:47:37,388] {standard_task_runner.py:52} INFO - Started process 9736 to run task
[2023-08-01 04:47:37,404] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_api_marvel_hacia_aws_redshift_multitask', 'extraer_data', 'scheduled__2023-08-01T04:45:35.277444+00:00', '--job-id', '917', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_multiple_tasks.py', '--cfg-path', '/tmp/tmpd5kcbvni', '--error-file', '/tmp/tmp7swzlwut']
[2023-08-01 04:47:37,406] {standard_task_runner.py:80} INFO - Job 917: Subtask extraer_data
[2023-08-01 04:47:37,702] {task_command.py:371} INFO - Running <TaskInstance: etl_api_marvel_hacia_aws_redshift_multitask.extraer_data scheduled__2023-08-01T04:45:35.277444+00:00 [running]> on host 24cbf2f01ed0
[2023-08-01 04:47:38,340] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=arielmfeldman.af@gmail.com
AIRFLOW_CTX_DAG_OWNER=Ariel Feldman
AIRFLOW_CTX_DAG_ID=etl_api_marvel_hacia_aws_redshift_multitask
AIRFLOW_CTX_TASK_ID=extraer_data
AIRFLOW_CTX_EXECUTION_DATE=2023-08-01T04:45:35.277444+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-01T04:45:35.277444+00:00
[2023-08-01 04:47:39,804] {python.py:173} INFO - Done. Returned value was: None
[2023-08-01 04:47:39,832] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=etl_api_marvel_hacia_aws_redshift_multitask, task_id=extraer_data, execution_date=20230801T044535, start_date=20230801T044737, end_date=20230801T044739
[2023-08-01 04:47:39,899] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-01 04:47:39,973] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-08-01 04:47:39,974] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-08-01 04:47:42,898] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
